04-materials/01-exploring-extensions.md

```diff
@@ -13,8 +13,19 @@
 :::
 
 
+## What is JupyterLab?
+
+It's not just an interface for working with Notebooks.
+The Jupyter Notebook interface has been around since _blank_, and in the mid-2010s,
```

Comment:
```suggestion
The Jupyter Notebook interface has been around since 2014, and in the mid-2010s,
```

Reply:
Woops, forgot to fill that in ;) A quick google suggests 2011 though. Is that right?